src/tracker/tracker-game-source.vala

```diff
@@ -0,0 +1,42 @@
+// This file is part of GNOME Games. License: GPLv3
+
+private class Games.TrackerGameSource : Object, GameSource {
+	private Tracker.Sparql.Connection connection { private set; get; }
+	private GenericSet<TrackerQuery> queries;
+
+	public TrackerGameSource (Tracker.Sparql.Connection connection) {
+		this.connection = connection;
+	}
+
+	construct {
+		queries = new GenericSet<TrackerQuery> (direct_hash, direct_equal);
```

Comment:
Well, I wouldn't prefix GLib stuff that is core part of the language, such as `Object` for example.
But those two were not even documented in the Vala tutorial, so it may be relevant.